packages/yarn-plugin-allow-scripts/package.json

```diff
@@ -10,19 +10,49 @@
   "license": "MIT",
   "private": true,
   "engines": {
-    "node": "^16.20.0 || ^18.0.0 || ^20.0.0 || ^22.0.0",
-    "yarn": "^3.1.1"
+    "node": "^18.0.0 || ^20.0.0 || ^22.0.0",
+    "yarn": "^4.4.1"
   },
-  "main": "./sources/index.ts",
+  "main": "./src/index.ts",
   "scripts": {
-    "build": "builder build plugin",
+    "build": "node scripts/build.js",
     "rebuild": "npm run build",
-    "test": "exit 0"
+    "test": "npm run build && ava"
   },
-  "dependencies": {
-    "@yarnpkg/builder": "3.3.1",
-    "@yarnpkg/cli": "3.8.5",
-    "@yarnpkg/core": "3.7.0",
-    "clipanion": "3.2.1"
+  "peerDependencies": {
```

Comment:
- It could still be used as a library, I guess
- More motivating: This communicates requirements to humans. I think it's a decent place to put authoritative version requirements for users even if tooling currently ignores it.

Reply:
with private:true there's no way this has any consequences, so it's fine